### PR TITLE
Add new 8.6.0 AWS wip version

### PR DESCRIPTION
**Pull request by corest**

### CODE REVIEW

File: aws.yaml [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+    - endpoint: http://cert-operator:8000
+      name: cert-operator
+      version: 0.1.0
+    - endpoint: http://chart-operator:8000
+      name: chart-operator
+      version: 0.7.0
+    - endpoint: http://cluster-operator:8000
+      name: cluster-operator
+      provider: aws
+      version: 0.20.0
```
Comment (MarcelMue) [inline aws.yaml]: Does 0.21.0 already exist?

Comment (corest) [reply aws.yaml]: I don't see it wired here https://github.com/giantswarm/cluster-operator/blob/master/service/controller/aws/legacy_cluster.go 
@rossf7 correct me if I'm wrong

Comment (rossf7) [reply aws.yaml]: I'm about to merge last PR to add 0.21.0. I just want to test it on ghost first.

https://github.com/giantswarm/cluster-operator/pull/685

Comment (rossf7) [reply aws.yaml]: Done.

Comment (corest) [reply aws.yaml]: updated, ptal

**Pull request merged**